mittab/apps/tab/outround_pairing_views.py

```diff
@@ -0,0 +1,639 @@
+        )))
+    # The minimum rank you want to warn on
+    warning = 5
+    excluded_people = list(
+        zip(*[
+            x + [""] * (size - len(x)) for x in [
+                list(excluded_teams),
+                list(excluded_judges),
+                list(non_checkins),
+                list(available_rooms)
+            ]
+        ]))
+
+    return render(request,
+                  "outrounds/pairing_base.html",
+                  locals())
+
+
+def alternative_judges(request, round_id, judge_id=None):
+    round_obj = Outround.objects.get(id=int(round_id))
+    round_gov, round_opp = round_obj.gov_team, round_obj.opp_team
+    # All of these variables are for the convenience of the template
+    try:
+        current_judge_id = int(judge_id)
+        current_judge_obj = Judge.objects.get(id=current_judge_id)
+        current_judge_name = current_judge_obj.name
+        current_judge_rank = current_judge_obj.rank
+    except TypeError:
+        current_judge_id, current_judge_obj, current_judge_rank = "", "", ""
+        current_judge_name = "No judge"
+
+    var_to_nov = TabSettings.get("var_to_nov", 2)
+    other_round_num = round_obj.num_teams / var_to_nov
+    if round_obj.type_of_round == BreakingTeam.NOVICE:
+        other_round_num = round_obj.num_teams * var_to_nov
+
+    other_round_type = BreakingTeam.NOVICE \
+                       if round_obj.type_of_round == BreakingTeam.VARSITY \
+                          else BreakingTeam.VARSITY
+
+    excluded_judges = Judge.objects.exclude(
+        judges_outrounds__num_teams=round_obj.num_teams,
+        judges_outrounds__type_of_round=round_obj.type_of_round
+    ).exclude(
+        judges_outrounds__num_teams=other_round_num,
+        judges_outrounds__type_of_round=other_round_type
+    ).filter(
+        checkin__round_number=0
+    )
+
+    query = Q(
+        judges_outrounds__num_teams=round_obj.num_teams,
+        judges_outrounds__type_of_round=round_obj.type_of_round
+    )
+    query = query | Q(
+        judges_outrounds__num_teams=other_round_num,
+        judges_outrounds__type_of_round=other_round_type
+    )
+
+    included_judges = Judge.objects.filter(query) \
+                                   .filter(checkin__round_number=0) \
+                                   .distinct()
+
+    excluded_judges = [(j.name, j.id, float(j.rank))
+                       for j in assign_judges.can_judge_teams(
+                           excluded_judges, round_gov, round_opp)]
+    included_judges = [(j.name, j.id, float(j.rank))
+                       for j in assign_judges.can_judge_teams(
+                           included_judges, round_gov, round_opp)]
+    included_judges = sorted(included_judges, key=lambda x: -x[2])
+    excluded_judges = sorted(excluded_judges, key=lambda x: -x[2])
+
+    return render(request, "pairing/judge_dropdown.html", locals())
+
+
+def alternative_teams(request, round_id, current_team_id, position):
```

Comment:
We should consider hiding this in the UI, has the possibility to really screw with things if people mis-click and mess up the seeding

Reply:
Important for switching gov and opp though.  I think this issue should exists (someone could manually mess up power pairing for example).  Better to give them the option and have them revert from backup if they really fuck up.  However, since it shows seeding in team cards, it's not that hard to fix.